UI/window-basic-main-transitions.cpp

```diff
@@ -1056,7 +1056,7 @@ QMenu *OBSBasic::CreatePerSceneTransitionMenu()
 		obs_data_set_int(data, "transition_duration", duration);
 	};
 
-	connect(duration, (void (QSpinBox::*)(int)) & QSpinBox::valueChanged,
+	connect(duration, (void(QSpinBox::*)(int)) & QSpinBox::valueChanged,
```

Comment:
This, and other similar changes, seem unrelated to the changes in this PR.

Reply:
Clang format messed this one up. Should be fixed now.